src/Core/Layer/LayerUpdateState.js

```diff
@@ -31,14 +54,23 @@ LayerUpdateState.prototype.canTryUpdate = function canTryUpdate(timestamp) {
         }
         case UPDATE_STATE.ERROR:
         default: {
+            const lastFailParams = this.lastFailureParams() || {};
+            const beforeLastFailParams = this.failureParams[this.failureParams.length - 2] || {};
+            // if two last Failure params aren't equivalent we can try without delay
```

Comment:
You can remove all this code because `secondsUntilNextTry` returns 0 in this case.